GeoDCAT charter edit (#109)
opengeospatial/geosemantics-dwg: geodcat_swg_charter/clause_3.adoc

The SWG will assess, based on the review of best practices and examples of use, if there is sufficient motivation to create GeoDCAT as an OGC Standard. 

The primary goal of this SWG's work is to define GeoDCAT as a set of modules or building blocks.

> The SWG will look into the relationships with other relevant standards and possibly provide conceptual mappings. Part of the GeoDCAT vocabulary deliverable will be the relationship with other relevant standards e.g., GeoDCAT-AP, ISO 19115, OGC API Records, STAC, GeoSPARQL, Dublin Core, schema.org, and others. 

Reviewer: ```suggestion
The SWG will look into the relationships with other relevant standards and possibly provide conceptual mappings. Part of the GeoDCAT vocabulary deliverable will be the relationship with other relevant standards, e.g., GeoDCAT-AP, ISO 19115, OGC API Records, STAC, GeoSPARQL, Dublin Core, schema.org, and others. 
```

The SWG will assess, based on the review of best practices and examples of use, if there is sufficient motivation to create GeoDCAT as an OGC Standard. 

The primary goal of this SWG's work is to define GeoDCAT as a set of modules or building blocks.

The SWG will look into the relationships with other relevant standards and possibly provide conceptual mappings. Part of the GeoDCAT vocabulary deliverable will be the relationship with other relevant standards, e.g., GeoDCAT-AP, ISO 19115, OGC API Records, STAC, GeoSPARQL, Dublin Core, schema.org, and others. 

> NOTE each of the standards mentioned above have their own lifecycle and governance. It is up to the SWG to decide if created mappings between GeoDCAT and these standards will be kept up to date. 

Reviewer: ```suggestion
**__NOTE:__** __Each of the standards mentioned above has its own lifecycle and governance. It is up to the SWG to decide whether mappings created between GeoDCAT and these standards will be kept up to date.__
```
